site2/website/release-notes.md

```diff
@@ -1,6 +1,300 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.10.0
+#### 2022-02-28
+
+### Important notice
```

Comment:
IMHO, should it be "Highlights" to introduce key enhancements? The "Importance Notice" section is more likely talking about notes that require attention, compatibility issues, etc.

Reply:
The items under this section are introduced default configuration changes or deployment changes, so it should be  "Importance Notice"

Reply:
Thanks for clarifying this. I suggest we add an introductory sentence like "This release introduces the following default configuration changes and deployment changes that require your attention."